lib/apiservers/portlayer/restapi/handlers/storage_handlers.go

```diff
@@ -292,9 +292,23 @@ func (handler *StorageHandlersImpl) CreateVolume(params storage.CreateVolumePara
 }
 
 //RemoveVolume : Remove a Volume from existence
-func (handler *StorageHandlersImpl) RemoveVolume(storage.RemoveVolumeParams) middleware.Responder {
+func (handler *StorageHandlersImpl) RemoveVolume(params storage.RemoveVolumeParams) middleware.Responder {
 	defer trace.End(trace.Begin("storage_handlers.RemoveVolume"))
-	return storage.NewRemoveVolumeOK() //TODO: this is just a stub for now.
+
+	_, err := storageVolumeLayer.VolumeGet(context.TODO(), params.Name)
```

Comment:
Won't the `destroy` give you an error if the given volume doesn't exist?  

I don't think we need this check since the destroy call itself will have to do more than just a check for existence anyway.